docs/reference/aggregations/metrics/boxplot-aggregation.asciidoc

```diff
@@ -0,0 +1,204 @@
+[role="xpack"]
+[testenv="basic"]
+[[search-aggregations-metrics-boxplot-aggregation]]
+=== Boxplot Aggregation
+
+A `boxplot` metrics aggregation that computes boxplot of numeric values extracted from the aggregated documents.
+These values can be extracted either from specific numeric fields in the documents, or be generated by a provided script.
+
+The `boxplot` aggregation returns essential information for making a https://en.wikipedia.org/wiki/Box_plot[box plot]: minimum, maximum
+median, first quartile (25th percentile)  and third quartile (75th percentile) values.
+
+==== Syntax
+
+A `boxplot` aggregation looks like this in isolation:
+
+[source,js]
+--------------------------------------------------
+{
+    "boxplot": {
+        "buckets_path": "my_cardinality_agg"
+    }
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+Let's look at a boxplot representing load time:
+
+[source,console]
+--------------------------------------------------
+GET latency/_search
+{
+    "size": 0,
+    "aggs" : {
+        "load_time_boxplot" : {
+            "boxplot" : {
+                "field" : "load_time" <1>
+            }
+        }
+    }
+}
+--------------------------------------------------
+// TEST[setup:latency]
+<1> The field `load_time` must be a numeric field
+
+The response will look like this:
+
+[source,console-result]
+--------------------------------------------------
+{
+    ...
+
+   "aggregations": {
+      "load_time_boxplot": {
+         "min": 0.0,
+         "max": 990.0,
+         "q1": 165.0,
+         "q2": 445.0,
+         "q3": 725.0
+      }
+   }
+}
+--------------------------------------------------
+// TESTRESPONSE[s/\.\.\./"took": $body.took,"timed_out": false,"_shards": $body._shards,"hits": $body.hits,/]
+
+As you can see, the aggregation will return a calculated value for each percentile
+in the default range.  If we assume response times are in milliseconds, it is
+immediately obvious that the webpage normally loads in 10-725ms, but occasionally
+spikes to 945-985ms.
+
+==== Script
+
+The boxplot metric supports scripting.  For example, if our load times
+are in milliseconds but we want percentiles calculated in seconds, we could use
```

Comment:
"percentiles" typo

Reply:
hmm, I am not sure how to call them here. Technically they are percentiles :) Values?

Reply:
Whoops, yeah this is probably fine.  Was thinking it was a copy/paste leftover.